hosts/azure/jenkins-controller/jenkins-casc.yaml

```diff
@@ -0,0 +1,126 @@
+# TODO: sort out jenkins authentication e.g.:
+# https://plugins.jenkins.io/github-oauth/
+# Below config requires admin to trigger builds or manage jenkins
+# allowing read access for anonymous users:
+
+jenkins:
+  agentProtocols:
+  - "JNLP4-connect"
+  - "Ping"
+  authorizationStrategy: "loggedInUsersCanDoAnything"
```

Comment:
instead of **loggedInUsersCanDoAnything** I think we should use more granular authorization strategy,  that restricts access based on user roles and permissions!

Please check https://plugins.jenkins.io/role-strategy/

Reply:
Yes, there's an internal ticket tracking this, but out of scope for this PR.